src/chaiVagueErrorsRule.ts

```diff
@@ -41,7 +41,12 @@ class ChaiVagueErrorsRuleWalker extends ErrorTolerantWalker {
     protected visitPropertyAccessExpression(node: ts.PropertyAccessExpression): void {
         if (ChaiUtils.isExpectInvocation(node)) {
             if (/ok|true|false|undefined|null/.test(node.name.getText())) {
-                this.addFailureAt(node.getStart(), node.getWidth(), FAILURE_STRING);
+                const expectInvocation: ts.CallExpression = ChaiUtils.getExpectInvocation(node);
+                if (expectInvocation && expectInvocation.arguments.length === 2) {
```

Comment:
Style nitpick: if you used `!expectInvocation || expectInvocation.arguments.length !== 2`, you could get rid of the `// do nothing` and `} else {` lines and just have this be one `if` statement. Not a blocker at all.